keps/prod-readiness/sig-network/1672.yaml

```diff
@@ -1,3 +1,7 @@
 kep-number: 1672
 alpha:
   approver: "@wojtek-t"
+beta:
```

Comment:
I really suggest splitting it into two PRs - I can easily imagine one being ready before the deadline and the other not being...

Reply:
Sure I will split the two PRs, just trying to avoid confusion because these two KEPs overlap quite significantly.